mdbx/cursor.go

```diff
@@ -168,7 +170,7 @@ func (c *Cursor) Get(setkey, setval []byte, op uint) (key, val []byte, err error
 	if op == Set {
 		key = setkey
 	} else {
-		if op != LastDup {
+		if op != LastDup && op != FirstDup {
```

Comment:
`||`? 

FYI: we have 2 versions - Erigon2 (devel) using version `v0.27.21`(mdbx-go branch v0_27), Erigon3 (e35) using version `v0.37.0`(mdbx-go branch master). 
You will need open same PR into v0_27 branch also.

Reply:
don't really understand why you have merge conflicts. likely you choose wrong "target branch" for pr?

Reply:
@AskAlexSharov thanks, you've predicted my question :)